### ISSUE TITLE
feature request

### ISSUE THREAD

**Issue by snaeil** [potamides/pantran.nvim#6]
Engine API calls often take long (>2 seconds).
It would be nice to see if pantran's API call is still pending or not.

For example if mode is `yank` I can not know when the API call is finished and keep pressing `p` until I know it's there.

It would be nice to get a message whenever the API call finished/aborted or a fancy loading spinner, or or or...

Thanks for this plugin, really appreciate it!

**Comment by potamides** [comment]
Yeah sounds like a very reasonable addition! I think we can implement this minimally, maybe by firing User autocmds. Then spinners etc could be implemented independently based on status line plugin, etc. I think I  can allocate some time for this at the end of the month. Until then pull requests are welcome!
